week 2/index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+<head>
+    <title>Currency Conversion</title>
+    
+</head>
+<body>
+    <div id="timer" class="timer-container"></div>
+    
+    <div class="form-container">
+        <h2>Insert a New Currency Rate</h2>
+        <button class="collapsible">Insert a New Currency Rate</button>
+        <div class="content">
+            <form id="insertCurrencyForm">
+                <label for="base">Base Currency:</label><br>
+                <input type="text" id="base" name="base" value="EUR" disabled><br>
+                <label for="targetCurrency">Target Currency:</label><br>
+                <input type="text" id="targetCurrency" name="targetCurrency"><br>
+                <label for="rate">Rate:</label><br>
+                <input type="number" step="0.00001" id="rate" name="rate"><br><br>
+                <input type="button" value="Insert Rate" onclick="insertCurrencyRate()">
+            </form>
+            <pre id="newRateOutput"></pre> 
+        </div>
+
+        <hr/>
+
+        <h2>Convert Currency</h2>
+        <button class="collapsible">Convert Currency</button>
+        <div class="content">
+            <form id="convertCurrencyForm">
+                <label for="fromCurrency">From:</label><br>
+                <input type="text" id="fromCurrency" list="currencySuggestions" oninput="filterSuggestions()" placeholder="Type to search...">
```

Comment:
The function `filterSuggestions` does not exist and it throws an error in the browser console.
<img width="347" alt="image" src="https://github.com/user-attachments/assets/5c74cc74-2ead-46f7-b2e8-eab7f52f442e">

Reply:
> Can I merge my previous PR now
Sure!